### DISCUSSION TITLE
希望可以增加支持自定义等宽字体的OpenType特性

### DISCUSSION THREAD

**Discussion by kanwakyuudai** [F9y4ng/GreasyFork-Scripts#175] [问答专区(Question / Answer)] [answered]
我在自定等宽字体里指定用JetBrainsMono，这个字体支持指定CSS样式font-feature-settings来选用不同风格的样式字图或者连字特性，详细见[How to enable OpenType features](https://github.com/JetBrains/JetBrainsMono/wiki/OpenType-features)

**Comment by F9y4ng** [comment, marked as answer]
等宽字体部分已在代码中预定义了font-feature-settings的部分常用属性，比如连体字属性、数字0的着重样式等，由于该属性的错误配置会引起样式异常，而一般用户又无法处理，所以暂时没有开放这方面的功能。

**Reply by kanwakyuudai** [reply]
我在油猴控制台里自己修改了源码里font-feature-settings对应字符串的值，可以展示我想要的效果了，但是这样覆盖源码的改动在更新时会丢失，还是希望在“设置自定义等宽字体”里新增一个字段，支持用户自行写入font-feature-settings，如果方便请您考虑一下。

**Reply by F9y4ng** [reply]
适合的时机我会增加这个小功能。
